app/models/application_record.rb

```diff
@@ -9,10 +9,16 @@ class ApplicationRecord < ActiveRecord::Base
     name_desc = @meta[:name_desc] || "Name of the #{@meta[:friendly_name] || @meta[:class_scope]}#{meta_example}"
     property :name, String, desc: name_desc
     property :present?, one_of: [true, false], desc: 'Object presence (always true)'
+    property :previous_revision, Object, desc: "Previous revision of the record as tracked in the audit, returns the same object if there is no previous revision"
```

Comment:
This will still be documented for all the records :thinking: I believe we could document this on `AssociationsChanges` I'm not sure if it would work, but it should IMHO.

Also, would `Object` -> `'Audited::Audit'` make sense to be more specific in docs? Object is very general return value and doesn't hold much information

Reply:
I believe documenting this on all record is the correct way, what is your concern?

About the Object, I am changing this to ApplicationRecord which makes more sense to me.